sycl/test/basic_tests/builtins_templates.cpp

```diff
@@ -224,9 +228,15 @@ void check() {
   GENFLOAT_CHECK(TWO_ARGS, distance)
   GENFLOAT_CHECK(ONE_ARG, length)
   GENFLOAT_CHECK(ONE_ARG, normalize)
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
```

Comment:
You don't need this check. This test only works under the breaking changes flag.